### PR TITLE
 show line separator for specefic params

### PR DESCRIPTION
**Pull request by souissimai**

### CODE REVIEW

File: src/components/dialogs/parameters/load-flow-parameters.js [hunk 1]
```diff
@@ -383,6 +383,7 @@ const SpecificLoadFlowParameters = ({
                             currentProvider
                         ],
                     }}
+                    showSeparator={true}
```
Comment (AAJELLAL) [inline src/components/dialogs/parameters/load-flow-parameters.js:386]: showSeparator is enough.

Comment (souissimai) [reply src/components/dialogs/parameters/load-flow-parameters.js]: done

**Review by AAJELLAL — COMMENTED** [review]
Test ok but when the dropdown  field is empty it takes up a very large space. Is  this the desired behavior?

**Review by AAJELLAL — APPROVED** [review]
Test ok.

**Review by Walid-Sahnoun — APPROVED** [review]

**Pull request merged**